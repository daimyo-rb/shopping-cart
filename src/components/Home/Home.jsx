import styles from "./Home.module.css";
import { Link } from "react-router-dom";

function Home() {
  return (
    <div className={styles.container}>
      <h1>Home</h1>
      <Link to="shop">Shop</Link>
    </div>
  )
}

export default Home