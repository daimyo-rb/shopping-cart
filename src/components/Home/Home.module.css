.container {
  display: flex;
  flex-direction: column;
  align-items: center;
}
.heroImg {
  width: 60%;
  margin-top: 10px;
}
.welcomeText {
  margin-top: 5px;
  font-size: 16px;
}