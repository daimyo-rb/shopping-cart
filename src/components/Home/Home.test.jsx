import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Home from "./Home";

describe("Home component", () => {
  it('renders Home component correctly (snapshot)', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});