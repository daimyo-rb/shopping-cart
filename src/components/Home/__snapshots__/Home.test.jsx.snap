// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home component > renders Home component correctly (snapshot) 1`] = `
<DocumentFragment>
  <div
    class="_container_d6eec2"
  >
    <img
      class="_heroImg_d6eec2"
      src="https://images.unsplash.com/photo-1625214214702-e2d227cfa495?q=80&w=874&auto=format&fit=crop&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
    />
    <p
      class="_welcomeText_d6eec2"
    >
      Welcome to my store!
    </p>
  </div>
</DocumentFragment>
`;
